### PR TITLE
Build: remove superfluous system-property hack

### PR DESCRIPTION
**Pull request by snazy**
NesQuEIT side of https://github.com/projectnessie/nessie/pull/7451

### CODE REVIEW

**Review by dimas-b — DISMISSED** [review]

**Comment by snazy** [comment]
Not sure why this fails in CI but not locally. Will investigate tomorrow.

File: .github/workflows/main.yml
Comment (XN137) [inline .github/workflows/main.yml:36]: do we want to merge this and remove it right after?

**Review by XN137 — DISMISSED** [review]

File: .github/workflows/main.yml
Comment (snazy) [reply .github/workflows/main.yml:36]: I'll remove it before merge

**Pull request merged**
